tools/pot/openvino/tools/pot/engines/ie_engine.py

```diff
@@ -323,10 +331,13 @@ def _process_dataset_async(self, stats_layout, sampler, print_progress=False,
         progress_log_fn = logger.info if print_progress else logger.debug
 
         # Load model to the plugin
-        executable_model = self._ie.load_network(network=self._model,
-                                                 device_name=self.config.device,
-                                                 num_requests=requests_num)
-        free_irs = executable_model.requests
+        executable_model = self._ie.compile_model(model=self._model, device_name=self.config.device)
```

Comment:
Now it looks like this is a compiled model. The object type is also called CompiledModel